portality/view/api_v1.py

```diff
@@ -40,8 +34,74 @@ def list_operations():
 def docs():
     return render_template('doaj/api_docs.html')
 
+@blueprint.route("/search/applications/<path:search_query>")
+@api_key_required
+def search_applications(search_query):
+    """
+    Search your applications
+    ---
+    tags:
+      - search
+    description: "Search your applications to the DOAJ"
+    parameters:
+      -
+        name: "search_query"
+        in: "path"
+        required: true
+        type: "string"
+      -
+        name: "page"
+        in: "body":
+        required: false
+        type: "int"
+      -
+        name: "pageSize"
+        in: "body"
+        required: false,
+        type: "int"
+      -
+        name: "sort"
+        in: "body"
+        required: false
+        type: "string"
+      -
+        name: "api_key"
```

Comment:
On the other hand it's probably going to be easier in the real world to let people tack it onto the URL than force them to subscribe to a particular best practice.